grr/parsers/linux_release_parser.py

```diff
@@ -87,6 +87,35 @@ def Parse(self):
     return complete, ParsedRelease(name, major, minor)
 
 
+class OsReleaseParseHandler(ReleaseParseHandler):
```

Comment:
I thought I put this comment in, but maybe it got lost. We definitely need a test case with an example file for this parser. I know you've copied the style of the one above but that one should probably have included a comment to the effect of "this is hacky and fragile, and needs to be rewritten properly" :) The parser is really tightly coupled to the exact current format of the file, if it changes slightly the code breaks, and then we probably need to support multiple formats somehow.

Since all we want to do here is pull out the name and version id I'd be inclined to just write regexes for those rather than attempting to parse the structure.